pdk/1.0/modules/ROOT/pages/policies-pdk-configure-features-jwt.adoc

```diff
@@ -436,7 +432,7 @@ async fn filter(
 }
 ----
 
-== Custom Claim Validation
+== Validate Custom Claims
```

Comment:
This section should be removed. I think @NachoRaik removed it from the source engineering document: https://docs.google.com/document/d/1dXpEO_53HhXV1GGBqJU0veLksWBMAPzrabOKnezygug/edit#heading=h.vymfgou3mfr